.changeset/cuddly-fireants-cheat.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/wonder-blocks-search-field": minor
```

Comment:
**nit:** This would probably account for a `patch` change, but I also understand if you want to keep it as `minor` as the API is being affected in some way (via the renderer HTML). Feel free to keep it or change it as you prefer.

Reply:
Ah yes thanks for catching that! I think `patch` makes sense since the external api (props) hasn't changed